gradle-plugin-code/src/main/java/com/trivago/cluecumber/CluecumberReportGradleTask.java

```diff
@@ -0,0 +1,371 @@
+package com.trivago.cluecumber;/*
```

Comment:
Isn't this just a duplicated class from the "original" Cluecumber? This should not be necessary if the core package has all functionality and is just reused by the Maven and Gradle implementations.

Reply:
Here I'm setting all the inputs, initializing also the properties and classes needed, that are handled in a different way compared to the Maven plugin, that class also extending the DefaultTask from Gradle. As also the logger used in that class is a different logger than the used into the Maven plugin.

Reply:
That's why I would pull everything into a core package and either use a standard logger that works with everything or let each version define their own logger that core then uses.

Reply:
Hi, yes, in that case, we have defined a Gradle logger (that uses the Gradle api logger) and a Maven logger (that uses the Maven plugin Logger), which is currently defined in each of the plugins, extending a class called BaseLogger present inside the Core.